### ISSUE TITLE
Install yumex-ng offline ?

### ISSUE THREAD

**Issue by Tuxman2** [timlau/yumex-ng#45]
Hello,

is there a way to install yumex-ng offline. I know that fedora copr is something similar to AUR. I always created package from AUR 
by downloading the PKGBUILD file and launched 'makepkg -s' command in order to be able to create and install the package on Arch Linux distributions and offline. Is there something similar under Fedora ?

Thanks.

**Comment by timlau** [comment]
No, there is not, don't see a good use-case for that, yumex is not useful without network connection, so why do you need a offline installation ?

**Comment by Tuxman2** [comment]
Hello,

I created a local mirror of Fedora. So, I can install or do updates for software offline. I tried yumex-ng with my local mirror and it works. So, I don't need internet connection to install packages. I like to install software offline but with Fedora Copr repository it is more difficult as it seems to be like AUR repository (source compilation). It is not possible to 'download' Fedora Copr repository like the others (Everything, Tainted, free, nonfree).

Regards.
